jquery.h5validate.js

```diff
@@ -161,6 +165,66 @@
 			}, validity);
 		},
 
+		/**
+		 * Add builtin validators to h5Validate. Currently, this adds the following validators:
+		 *   Required
+		 *   Maxlength
+		 *   Pattern
+		 * @param  {object}  settings   instance settings
+		 */
+		addBuiltinValidators = function (settings) {
```

Comment:
Absolutely! This pull request does treat them all the same way - the only difference here is that the "addBuiltinValidators" function is adding validators that are shipped with h5Validate (and therefore could be said to be, in one sense of the word, "builtin"). On the other hand, the function name could certainly be changed to something like "addValidators" or similar so as not to miscommunicate its purpose. Please advise